features/support/env.rb

```diff
@@ -3,3 +3,75 @@
 Before do
   @aruba_timeout_seconds = 10 # too slow on sloppy machines
 end
+
+After do
+  FileUtils.rm_rf('tmp')
+end
+
+def disable_linters
+  <<-content
+  AvoidOutlineForSingleExample:
```

Comment:
I added the disable_all method based on @lindt's comment re: the specific test.  I can go ahead and add this to all the feature files.  @lindt, what do you think?